packages/clerk-js/src/ui/utils/errorHandler.ts

```diff
@@ -15,10 +15,17 @@ function setFieldErrors(fieldStates: Array<FormControlState<string>>, errors: Cl
   }
 
   fieldStates.forEach(field => {
-    const error = errors.find(err => {
+    let errorCallback = field?.callback;
```

Comment:
🔧 We should also rename this variable. `errorCallback` hints a function that runs when erroring. This is a function that manipulates the errors and returns the final error string.